tests/dataset/test_study_locus.py

```diff
@@ -199,8 +213,13 @@ def test_annotate_credible_sets(
     )
     data_sl = StudyLocus(
         _df=spark.createDataFrame(observed, schema), _schema=StudyLocus.get_schema()
-    ).annotate_credible_sets()
```

Comment:
How did it work before? It seems the same method is called on data_sl object twice...